src/Rector/Contrib/PHPUnit/DelegateExceptionArgumentsRector.php

```diff
@@ -68,19 +78,12 @@ public function refactor(Node $methodCallNode): ?Node
         $identifierNode = $methodCallNode->name;
         $oldMethodName = $identifierNode->name;
 
-        $this->prependNewMethodCall(
-            $methodCallNode,
-            $this->oldToNewMethod[$oldMethodName],
-            $methodCallNode->args[1]
-        );
+        $this->prependNewMethodCall($methodCallNode, $this->oldToNewMethod[$oldMethodName], $methodCallNode->args[1]);
         unset($methodCallNode->args[1]);
 
+        // prepend exception code method call
         if (isset($methodCallNode->args[2])) {
-            $this->prependNewMethodCall(
-                $methodCallNode,
-                'expectExceptionCode',
-                $methodCallNode->args[2]
-            );
+            $this->prependNewMethodCall($methodCallNode, 'expectExceptionCode', $methodCallNode->args[2]);
```

Comment:
There should be Fixer for this :smile: 

 Follow up to:

https://github.com/Symplify/CodingStandard#arguments-should-be-on-the-samestandalone-line-to-fit-line-length

Reply:
This is help us save some lines of code :pray: